lib/web/join_tokens_test.go

```diff
@@ -985,6 +934,58 @@ func TestIsSameAzureRuleSet(t *testing.T) {
 	}
 }
 
+func TestJoinScriptEnterprise(t *testing.T) {
```

Comment:
Is this just a move with a `t.Parallel()` and `const` added? If there are no significant changes I'd keep it where it was, so line blame doesn't change for little reason.